packages/appstore/src/components/modals/wallet-modal/wallet-modal.tsx

```diff
@@ -3,15 +3,22 @@ import { Modal } from '@deriv/components';
 import WalletModalHeader from './wallet-modal-header';
 import WalletModalBody from './wallet-modal-body';
 import { observer, useStore } from '@deriv/stores';
+import { useWalletList } from '@deriv/hooks';
 
 const WalletModal = observer(() => {
+    const { data } = useWalletList();
     const store = useStore();
     const {
         client: { balance, currency, landing_company_shortcode: shortcode },
         ui: { is_dark_mode_on, is_wallet_modal_visible, is_mobile, setIsWalletModalVisible },
         traders_hub: { is_demo },
     } = store;
 
+    // TODO: This should be removed when we refactor how to pass data
+    const current_wallet = data?.find(wallet =>
```

Comment:
@hamid-deriv just to be sure, this will be refactored to use @thisyahlen-deriv's `useActiveWallet` hook when it's merged, right?

Reply:
Yes, I added a TODO for this.